.travis.yml

```diff
@@ -1,17 +1,35 @@
 conditions: v1
 
-os: linux
-dist: trusty
-sudo: false
-group: edge
+matrix:
+    include:
+        - os: linux
+          sudo: false
+          group: edge
+          dist: trusty  
+          python: 2.7
+        - os: linux
+          sudo: false
+          group: edge
+          dist: trusty
+          python: 3.4
+        - os: linux
+          sudo: false
+          group: edge
+          dist: trusty
+          python: 3.5
+        - os: linux
+          sudo: false
+          group: edge
+          dist: trusty
+          python: &mainstream_python 3.6
+        - os: linux
+          sudo: false
+          group: edge
+          dist: trusty
+          python: nightly
+        - os: osx
```

Comment:
This will not work out of the box.
Here's the best way to make it right: https://github.com/aio-libs/aiohttp/blob/1e74706/.travis.yml#L82-L108. There's another way which is to use `pyenv` but it's slow so I'm gradually replacing it with downloading installers from the official web site.